lib/loader/loader.js

```diff
@@ -252,7 +252,11 @@ var loader, define, requireModule, require, requirejs;
     this.name = path;
   }
 
-  define.alias = function(path) {
+  define.alias = function(path, target) {
+    if (arguments.length === 2) {
```

Comment:
I think we can avoid using arguments here by just checking that target !== undefined (or typeof target === "string")